### DISCUSSION TITLE
serde-human-readable info

### DISCUSSION THREAD

**Discussion by Geobert** [time-rs/time#372] [Question] [answered]
Hi,

I'm not sure to understand what is this `serde-human-readable` feature flag.

What is supported? 

Is this as powerful as `htp`?

**Comment by jhpratt** [comment, marked as answer]
From the docs:

> Allows serde representations to use a human-readable format. This is determined by the serializer, not the user. If this feature is not enabled or if the serializer requests a non-human-readable format, a format optimized for binary representation will be used.

Basically, is the format being displayed to a human or only used internally? If it's being shown to a person, enable this flag. If not, it's needless overhead.

**Reply by Geobert** [reply]
I see, thank you very much!
